scripts/yocto-build-and-test.sh

```diff
@@ -145,7 +145,7 @@ EOF
     if has_component mender-gateway; then
         local mender_gateway_filename=$(find $WORKSPACE/stage-artifacts/ -maxdepth 1  -name "mender-gateway-*.tar.xz" | head -n1 | xargs basename)
         tar -C /tmp -xf $WORKSPACE/stage-artifacts/$mender_gateway_filename ./${mender_gateway_filename%.tar.xz}/x86_64/mender-gateway
-        local mender_gateway_version=$(/tmp/${mender_gateway_filename%.tar.xz}/x86_64/mender-gateway --version | egrep -o '[0-9]+\.[0-9]+\.[0-9b]+(-build[0-9]+)?')
+        local mender_gateway_version=$(/tmp/${mender_gateway_filename%.tar.xz}/x86_64/mender-gateway --version | head -1 | egrep -o '[0-9]+\.[0-9]+\.[0-9b]+(-build[0-9]+)?' | head -1)
```

Comment:
Nitpick: I think `head -n 1` is preferred. The `-1` is not even documented, AFAICS.

Reply:
> Maybe add a short explanation, and in the long. run we should match on the leading string. For now to get the release proceeding, okay with me.

I agree actually, we're not in that much of a hurry, let's do it proper. I'll match on the line we want.

Reply:
Taken to school as usual :school: